scripts/setup.sh

```diff
@@ -26,7 +26,9 @@ if [ "${SKIP_CYPHERNODE}" == "true" ]; then
 elif [ "${CFG_PASSWORD}" != "" ]; then
   echo 'Building Cyphernode' && \
   # Install cyphernode using premade config
-  CFG_PASSWORD=$CFG_PASSWORD ./cyphernode/setup.sh -irc
+  cd cyphernode/
```

Comment:
Re-use the behaviour of `scripts/stop.sh` and add a variable to move in and out of cyphernode's folder. This will ensure we can call the script from anywhere. (super important for production)

Reply:
right good catch